plutus-example/test/plutus-example-test.hs

```diff
@@ -22,6 +24,7 @@ plutusExampleTests = testGroup "plutus-example"
    , testProperty "prop_ScriptData_MyCustomRedeemer_JSON" Test.PlutusExample.ScriptData.prop_ScriptData_MyCustomRedeemer_JSON
    -- TODO: Re-enable these when plutus-apps is bumped to the latest cardano-node
```

Comment:
Should we re-enable as we bump to the latest cardano-node?

Reply:
Tried to, but the tests don't compile. Didn't try to debug why, as it's out of scope for this PR.